vendor/go.jolheiser.com/pwn/pwn.go

```diff
@@ -0,0 +1,55 @@
+package pwn
+
+import (
+	"context"
+	"io"
+	"net/http"
+)
+
+const userAgent = "Go-Pwn"
```

Comment:
I know this is feedback for an upstream library, but perhaps you could put a version number in here, as well as a link to go.jolheiser.com/pwn so that should HIBP needed to provide feedback to the library then they could say version x is causing issues, and also know where to report the issues. Lmk if I should create an issue for this in your library?

Reply:
A good idea. You can create an issue if you want, I should be able to update it later today.

Reply:
Done. :slightly_smiling_face: